app/src/main/java/org/mozilla/fenix/theme/FirefoxColors.kt

```diff
@@ -0,0 +1,140 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.theme
+
+import androidx.compose.ui.graphics.Color
+
+/**
+ * All standardized colors for Firefox applications.
+ *
+ * For a consistent UX all composables should use these colors only.
+ *
+ * [v 0.1.0 from September 8 2021](https://www.figma.com/file/DyIIHvRgqt2EXfAVn1gV9c/Fx-Colors)
+ */
+@Suppress("MagicNumber")
+object FirefoxColors {
+    val Black = Color(0xFF000000)
+    val White = Color(0xFFFFFFFF)
+
+    // Light grey should primarily be used for the Light Theme and secondary buttons.
+    val LightGrey05 = Color(0xFFFBFBFE)
+    val LightGrey10 = Color(0xFFF9F9FB)
+    val LightGrey20 = Color(0xFFF0F0F4)
+    val LightGrey30 = Color(0xFFE0E0E6)
+    val LightGrey40 = Color(0xFFCFCFD8)
+    val LightGrey50 = Color(0xFFBFBFC9)
+    val LightGrey60 = Color(0xFFAFAFBA)
+    val LightGrey70 = Color(0xFF9F9FAD)
+    val LightGrey80 = Color(0xFF8F8F9D)
+    val LightGrey90 = Color(0xFF80808E)
+
+    // Dark grey should primarily be used for the Dark theme and secondary buttons.
```

Comment:
There's already `PhotonColors` provided by `ui-colors`.

Reply:
I can move these to AC.

Reply:
This is actually in AC already as PhotonColors.